lib/ansible/modules/network/cloudengine/ce_vxlan_vap.py

```diff
@@ -707,9 +703,9 @@ def config_vap_sub_intf(self):
 
     def config_vap_vlan(self):
         """configure a VLAN as a service access point"""
-
-        if not self.vap_info:
-            self.module.fail_json(msg="Error: Bridge domain %s does not exist." % self.bridge_domain_id)
+        #
+        # if not self.vap_info:
```

Comment:
Please don't commit commented code. If it's unnecessary, remove it, and if it's broken, fix it.